cli/README-cn.md

```diff
@@ -0,0 +1,718 @@
+
+| 平台                         | 准备命令（Prepare command ） | 包的路径 (相对项目的根目录) |
+|-------------------------------------|--------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| Cordova (Android)             | `cordova prepare android`                    | `./platforms/android/assets/www` 目录                                                                 |
+| Cordova (iOS)                    | `cordova prepare ios`                                  | `./platforms/ios/www ` 目录                                                                      |
+| React Native wo/assets (Android) | `react-native bundle --platform android --entry-file <entryFile> --bundle-output <bundleOutput> --dev false`                                               |  `--bundle-output` 参数的值                                                                      |
+| React Native w/assets (Android)  | `react-native bundle --platform android --entry-file <entryFile> --bundle-output <releaseFolder>/<bundleOutput> --assets-dest <releaseFolder> --dev false` |  `--assets-dest` 参数的值，应该是一个包含资源和JS bundle的新创建的目录。|
+| React Native wo/assets (iOS)    | `react-native bundle --platform ios --entry-file <entryFile> --bundle-output <bundleOutput> --dev false`   | `--bundle-output` 参数的值  |
+| React Native w/assets (iOS)      | `react-native bundle --platform ios --entry-file <entryFile> --bundle-output <releaseFolder>/<bundleOutput> --assets-dest <releaseFolder> --dev false` | `--assets-dest` 参数的值，应该是一个包含资源和JS bundle的新创建的目录。|
+
+#### Target binary version (目标二进制版本)参数
+
+这是你想发布更新的特定仓库/二进制版本，这样只有那个版本上的用户才会接收到更新，而那些运行较老/新版本用户则不会。这样很有用，原因如下：
+
+1. 如果有用户运行一个很老的版本，有可能在CodePush的更新里有个破坏性的更新，这跟他们现在运行的版本不兼容。
+
+2. 如果用户正在运行一个新的二进制版本，那么假定，他们正在运行并更新CodePush 更新（可能不兼容）。
+
+如果你想更新应用商店里二进制文件的多个版本，我们允许你指定参数像这样[语义版本范围表达式](https://github.com/npm/node-semver#advanced-range-syntax)。这样， 任何在版本号范围内（如：`semver.satisfies(version, range)` returns `true`）的客户端设备都能获得更新。
+
+如下是有效的版本号范围表达式的例子：
+
+| 范围表达式 | 谁获得更新                                                                    |
+|------------------|----------------------------------------------------------------------------------------|
+| `1.2.3`          | 只有`1.2.3`版本  |
+| `*`                 | 所有版本                    |
+| `1.2.x`          | 主版本为1，小版本为2的任何版本  |
+| `1.2.3 - 1.2.7`  | 在 `1.2.3` (包含) 和 `1.2.7` (包含) 之间的版本 |
+| `>=1.2.3 <1.2.7` | 在 `1.2.3` (包含) 和 `1.2.7` (不包含)之间的版本 |
+| `~1.2.3`         | 相当于`>=1.2.3 <1.3.0`                                                         |
+| `^1.2.3`         | 相当于`>=1.2.3 <2.0.0`                                                         |
+
+*注意：如果语义表达式以特殊字符开始如`>`,`^`或***，如果你没有用引号括起来的话命令可能执行不对，因为shell在CLI里不支持右边的值。所以，当调用`release`命令时最好能把你的`targetBinaryVersion`参数用双引号括起来，如：`code-push release MyApp updateContents ">1.2.3"`。*
+
+如下表格分别概括了每个应用类型的CodePush更新的语义版本范围的版本值：
+| 平台               | 应用商店版本来源 |
```

Comment:
Add an empty line above this line. The formatting is jumbled up in the rendered doc.